lib/web_ui/lib/src/engine/window.dart

```diff
@@ -396,7 +396,6 @@ class EngineWindow extends ui.Window {
             return;
```

Comment:
The one above also has the same problem (I can't comment on the line above :P ).

Reply:
Yes! Set data(copy) works, but if it doesn't work, it will always return "true" :) removing it as well.

Reply:
done